source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to sync data between a replica set and a 
+sharded cluster using :program:`mongosync`. To perform a sync between two
+replica sets, see :ref:`c2c-quickstart`.
```

Comment:
[question] Does the source have to be a RS and the destination have to be a shared cluster? Or can it also go the other way around?

Reply:
It cannot be the other way around. I will clarify this.